en/datenshi_beatmap_criteria_en.md

```diff
@@ -7,7 +7,7 @@ A few question that are frequently asked
 
 -----------------------
 
-- **Q: My map hasn't been ranked but it follows the criterea?**
+- **Q: My map hasn't been ranked but it follows the criteria?**
 - A: There's a chance that your map doesn't get detected by the system, you can re-request again in discord with this format : `[REQ](osu-beatmap-link)(mode)` **Only for Donaters**
```

Comment:
```suggestion
A: There's a chance that your map doesn't get detected by the system, you can re-request again in discord channels `#beatmap-requests` **Only for Donaters**
```
ganti ini

Reply:
bisa diganti dengan
'you can re-request again in the discord channel #beatmap-requests'
gk pake s karena cuman satu channel